Fastfile.private

```diff
@@ -155,42 +157,65 @@ reflected in `Info.plist` during building."
   desc "It uses the same password stored in keychain for the current user."
   desc "Make sure to store safely the output of this command in the right `Google Drive` folder."
   private_lane :generate_push_certificates do |options|
-    build_configuration = options[:build_configuration]
+
+    build_configuration_key = options[:build_configuration]
+    build_configuration = get_build_configuration(build_configuration: build_configuration_key)
+    match_type = get_match_type(build_configuration: build_configuration_key)
```

Comment:
Should this receive the key or the enum?

Reply:
As it's currently written it receives the symbol. We can agree to change it. I'm not sure it's the best approach because we some times provide symbols and some times strings, and it may be confusing.